sdks/python/apache_beam/dataframe/frames_test.py

```diff
@@ -1100,6 +1100,44 @@ def test_dt_tz_localize_nonexistent(self):
             'Europe/Warsaw', ambiguous='NaT', nonexistent=pd.Timedelta('1H')),
         s)
 
+  def test_idxmin(self):
+    df = pd.DataFrame({
+        'consumption': [10.51, 103.11, 55.48],
+        'co2_emissions': [37.2, 19.66, 1712]
+    },
+                      index=['Pork', 'Wheat Products', 'Beef'])
+
+    s = pd.Series(data=[4, 3, None, 1], index=['A', 'B', 'C', 'D'])
+    s2 = pd.Series(data=[1, 2, 3], index=[1, 2, 3])
+
+    self._run_test(lambda df: df.idxmin(), df, nonparallel=True)
```

Comment:
It looks like you were able to make `Series.idxmin` parallelizable by breaking it up into two steps. Shouldn't you be able to do the same thing for `DataFrame.idxmin` with `axis=0`?

Reply:
Yes, you're right 👍 . I already pushed this change and Dataframe can now be parallelized with any `axis`.